content/php/concepts/math-functions/tan/tan.md

```diff
@@ -0,0 +1,30 @@
+#tan()
+
+PHP Calculate Tangent
```

Comment:
Remove this line. It is another title header, which is unnecessary. Also remove the extra line breaks so there's no more than one blank line in a row